web/src/games/cardtable/CardContainer.tsx

```diff
@@ -15,6 +16,7 @@ interface ICardContainerProps {
       handleRotateTurn: () => void;
     };
     gameState: {
+      playerID?: string;
       stage?: string;
```

Comment:
You could replace the string with a enum, so you get compiler type safety instead of comparing strings.

Reply:
Same thing for phase

Reply:
addressed in upcoming commit -- added playerEnum, stageEnum, phaseEnum, cardEnum